content/post/house-prices-in-king-county.Rmd

```diff
@@ -290,19 +312,21 @@ at = AutoTuner$new(lrn_xgb, cv3, measures = msr("regr.mse"), ps,
   terminator, tuner = tnr("random_search"))
 ```
 
-```{r, echo = FALSE, results = 'hide', eval = FALSE}
+<!-- The following 2 snips are not evaluated because of runtime, no idea how we want to handle this.-->
```

Comment:
How long does it take? I'd really be in favor of only including evaluated chunks. We had problems with non-eval chunks in the mlr-blog already.

Especially if things change in mlr3 we need to recognize this.

Reply:
I included tuning and made it faster. Takes ~10 seconds now, which is allright I guess

Reply:
Great!